### ISSUE TITLE
EML files with attachments show NAN for sizes

### ISSUE THREAD

**Issue by querixg** [cleidigh/printing-tools-ng#125]
Linux Mint 20.1
Thunderbird 91.9.1
PTNG 2.1.1

@cleidigh

1. If a mail has been saved as a file (.eml) and then is opened by Thunderbird, PTNG lists all attachments in the header with „NaN undefined‟ instead of their size.

2. If a mail is forwarded as attachment, it is attached as an eml file. If the recipient opens this attachment, PTNG does not work at all, Thunderbird‛s normal print does. In order to use PTNG, the attachment first has to be saved as a file and then the file be opened by Thunderbird.

I don‛t ask for a fix, I just wanted to let you know.

For my purposes PTNG is great!

**Comment by cleidigh** [comment]
@querixg 
Good catch! This also made me find a new problem with 102.
b7 fixes #1 the other I think I will wait on. I am working on other 102 releases
@cleidigh

**Comment by querixg** [comment]
@cleidigh
The new title covers my first observation only. Since I described a workaround for the second (save the attachment as file first), it may be left open for a long time, but should not disappear. Therefore I  created a new issue and I'll close this one.

**Comment by cleidigh** [comment]
@querixg 
I assume it tested ok?

**Comment by querixg** [comment]
@cleidigh
Oh, I trusted your statement. I'm happy with v2.1.1.

**Comment by querixg** [comment]
@cleidigh
I just confirmed that the sizes of attachments of mails opened from eml files are listed when using v2.1.3.
